Packages/com.unity.inputsystem/Documentation~/UISupport.md

```diff
@@ -13,14 +13,31 @@ uid: input-system-ui-support
 * [UI and game input](#ui-and-game-input)
 * [UI Toolkit support](#ui-toolkit-support)
 
-You can use the Input System package to control any in-game UI created with the [Unity UI package](https://docs.unity3d.com/Manual/UISystem.html). The integration between the Input System and the UI system is handled by the [InputSystemUIInputModule](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html) component.
+You can use the Input System package to control any in-game UI created with the [Unity UI package](https://docs.unity3d.com/Manual/UISystem.html).
+
+
+⭐ From **Unity 2023.2.x** and newer, the Input System and UI Toolkit are fully integrated. To configure UI Input you can
+configure the UI action map of project-wide actions in **[Project Settings > Input System Package > Input Actions](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/Workflow-Actions.html)**.
```

Comment:
I think this should be a relative URL so that it references the latest for future releases (rather than always pointing at 1.8)